documentation/builders/installation.md

```diff
@@ -10,7 +10,7 @@ Before you can install the Phoniebox software, you need to prepare your Raspberr
 1. Connect a Micro SD card to your computer (preferable an SD card with high read throughput)
 2. Download the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) and run it
 3. Click on "Raspberry Pi Device" and select "No filtering"
-4. Select **Raspberry Pi OS Lite (32-bit)** (without desktop environment) as the operating system. `future3` does not support 64bit kernels (`aarch64`).
+4. Select **Raspberry Pi OS Lite (32-bit)** (without desktop environment) as the operating system. *64-bit is currently not supported.* For Pi 4 and newer also check [this](#workaround-for-64-bit-kernels-pi-4-and-newer).
```

Comment:
Maybe we should suggest to choose "Bullseye" for now. It's an LTE and will be around for a while, especially for lower end Pi's like Pi 1, 2, 3 and Zero/2. Even the Pi Imager does not recommend to install Bookworm on those devices.